src/actions/userActions.js

```diff
@@ -21,4 +23,25 @@ const registerUser = (postData) => dispatch => {
     });
 };
 
-export default registerUser;
+export const userLogin = postData => dispatch => {
+  toast.dismiss();
+  dispatch(loginUserInitiated(true));
+  axiosInstance
+    .post('/api/v1/auth/login', postData)
+    .then(response => {
+      localStorage.setItem('authToken', response.data.message.access_token);
+      dispatch(loginUserSuccess(true));
+      toast.success(
+        response.data.message.message,
+        { autoClose: 3500, hideProgressBar: true },
+        { position: toast.POSITION.TOP_CENTER },
+      );
+    })
+    .catch(error => {
+      toast.error(
+        error.response.data.message.message,
```

Comment:
Great work @mendozabree 👍
I do not see you dispatching something here to turn the loader off in the error section.

Reply:
Oh yeah! Thanks for pointing that out let me add that